samples/subsys/display/lvgl/src/main.c

```diff
@@ -17,16 +17,16 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(app);
 
+#define IS_LV_POINTER_ENABLED                                                                      \
+	(IS_ENABLED(CONFIG_LV_Z_POINTER_KSCAN) || IS_ENABLED(CONFIG_LV_Z_POINTER_INPUT))
+
 static uint32_t count;
 
 #ifdef CONFIG_GPIO
-static struct gpio_dt_spec button_gpio = GPIO_DT_SPEC_GET_OR(
-		DT_ALIAS(sw0), gpios, {0});
+static struct gpio_dt_spec button_gpio = GPIO_DT_SPEC_GET_OR(DT_ALIAS(sw0), gpios, {0});
```

Comment:
Hey, format changes not related to the functionality of the commit should be avoided, this is a small patch so it's not that important but in general the commit should be isolated to one change so it's easier to review. Could you avoid changing these?

Reply:
Understood. My editor automatically applies clang-format and I had a separate commit with the format changes.
formatting changes have been reverted.